scripts/pre_commit_linter.py

```diff
@@ -85,6 +85,18 @@
         'Please make sure all files only have LF endings (no CRLF).')
 }
 
+BAD_PATTERNS_JS = {
+    ' == ': (
+        'Please replace == with === in this file.'),
+    ' != ': (
+        'Please replace != with !== in this file.')
+}
+
+EXCLUDED_FILES_JS = (
```

Comment:
We should only exclude these from the ' == ' check, and not from other patterns.

A nice declarative way to do this is to change the schema of BAD_PATTERNS and BAD_PATTERNS_JS. Instead of the value being a string, it could be a dict instead. The dict would have the keys 'message' (which contains the current message) and 'excluded_files' (which contains a list of excluded files). I think this is better design, and more easily extensible in the future.

Otherwise, everything looks good!